src/mono/sample/wasm/console-v8-es6/v8shim.cjs

```diff
@@ -0,0 +1,15 @@
+if (typeof globalThis.URL === 'undefined') {
+    globalThis.URL = class URL {
+        constructor(url) {
+            this.url = url;
+        }
+        toString() {
+            return this.url;
+        }
+    };
+}
+
+import('./main.mjs').catch(err => {
+    console.log(err);
```

Comment:
i assume we're not using console.error here because console v8 doesn't implement it?

Reply:
right